spec/features/record_show_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe GenericFile do
+  context 'record', :type => :feature do
+
+    let(:user) { FactoryGirl.find_or_create :user_with_fixtures }
+    let!(:file) do
+      GenericFile.new.tap do |f|
+        f.title = ['little_file.txt']
+        f.creator = ['little_file.txt_creator']
+        f.resource_type = ["Thesis" ]
+        f.read_groups = ['public']
+        f.abstract = "This is a <a href=\"https://library.ualberta.ca\">test link</a>"
+        f.is_version_of = "This is <b>bold</b> text."
+        f.apply_depositor_metadata(user.user_key)
+        f.save!
+      end
```

Comment:
There's a lot of setup here that isn't necessary:
```
diff --git a/spec/features/record_show_spec.rb b/spec/features/record_show_spec.rb
index 1c57371..6ca999c 100644
--- a/spec/features/record_show_spec.rb
+++ b/spec/features/record_show_spec.rb
@@ -6,10 +6,6 @@ describe GenericFile do
     let(:user) { FactoryGirl.find_or_create :user_with_fixtures }
     let!(:file) do
       GenericFile.new.tap do |f|
-        f.title = ['little_file.txt']
-        f.creator = ['little_file.txt_creator']
-        f.resource_type = ["Thesis" ]
-        f.read_groups = ['public']
         f.abstract = "This is a <a href=\"https://library.ualberta.ca\">test link</a>"
         f.is_version_of = "This is <b>bold</b> text."
         f.apply_depositor_metadata(user.user_key)
@@ -21,12 +17,8 @@ describe GenericFile do
       cleanup_jetty
     end
 
-    before do 
-      sign_in user 
-      visit "/files/#{file.id}"
-    end
-   
     it "page should not have html tags" do
+      visit "/files/#{file.id}"
       expect(page).to have_content('This is a test link')
       expect(page).to have_link('test link', href: 'https://library.ualberta.ca')
       parent = page.find("span[itemprop='is_version_of']")
```

I'm also seeing that this test takes 13 seconds to run.